frontend/src/home/Home.tsx

```diff
@@ -457,30 +388,16 @@ class HomeComponent extends React.Component<Props, HomeState> {
 
   logOut = async () => {
     await this.updatePlan(false);
-    this.props.logOut();
     removeAuthTokenFromCookies();
 
     alert(
       "Your plan has been updated and you have been logged out. You will be redirected to the welcome screen."
     );
     this.props.history.push("/");
+    this.props.logOut();
```

Comment:
We need to do this after we dismount Home.

Reply:
Just curious, why is this?

Reply:
when we clear stuff from redux, we break the home component